internal/jet/func_expression.go

```diff
@@ -843,3 +855,60 @@ func newTimestampzFunc(name string, expressions ...Expression) *timestampzFunc {
 func Func(name string, expressions ...Expression) Expression {
 	return NewFunc(name, expressions, nil)
 }
+
+type rangeFunc struct {
+	funcExpressionImpl
+	rangeInterfaceImpl
+}
+
+// NewRangeFunc creates new range function with name and expression parameters
+func NewRangeFunc(name string, expressions ...Expression) *rangeFunc {
+	rangeFunc := &rangeFunc{}
+
+	rangeFunc.funcExpressionImpl = *NewFunc(name, expressions, rangeFunc)
+	rangeFunc.rangeInterfaceImpl.parent = rangeFunc
+
+	return rangeFunc
+}
+
+func NumRange(lowNum, highNum NumericExpression, bounds ...StringExpression) RangeExpression {
+	if len(bounds) != 2 {
+		panic("bound must contains 2 elements for lower and higher bound")
+	}
+	return RangeExp(NewFunc("numrange", []Expression{lowNum, highNum, bounds[0], bounds[1]}, nil))
```

Comment:
I don't think this will work. 3rd parameter is single string expression. So you just need to pass `,bounds...`.   
The only condition for bounds length is that it can't be greater then 1.